core/templates/base-components/base-content.component.html

```diff
@@ -51,7 +51,7 @@
                 <oppia-top-navigation-bar [headerText]="getHeaderText()"
                                           [subheaderText]="getSubheaderText()">
                 </oppia-top-navigation-bar>
-                <div class="collapse d-lg-inline-flex navbar-collapse oppia-navbar-collapse ng-cloak navbar-options-display">
+                <div class="collapse d-inline-flex navbar-collapse oppia-navbar-collapse ng-cloak navbar-options-display">
```

Comment:
Actually, could you explain why we are modifying base-content component? This is used in many places I think so it is important that we ensure the change only affects the translation page in the editor.